.azure/release.yml

```diff
@@ -43,12 +38,12 @@ stages:
         64Bit:
           arch: x86_64
           plat: manylinux1_x86_64
```

Comment:
Sorry for not seeing it right there :roll_eyes:

Reply:
I agree on splitting the job, but not necessarily for the same reasons...


I was going with the fact that ``manylinux1`` is now obsolete, and even ``manylinux2010`` is becoming dated. Choosing to release ``JPype`` with ``Python >=3.7`` seems to be compatible with effectively providing ``CentOS7`` and beyond compatible wheels (users can always compile their own wheels if they really want older glibc compat).